registries/adapters/openshift_adapter_test.go

```diff
@@ -28,6 +28,8 @@ import (
 )
 
 var Images = []string{"satoshi/nakamoto", "foo/bar", "paul/atreides"}
+var TestURL = "https://registry.reg-aws.openshift.com:443"
+var RegistryName = "registry.reg-aws.openshift.com:443"
```

Comment:
No need to declare these strings here. I would localize them to the test case instead.